README.md

```diff
@@ -11,7 +11,8 @@
 - HTML & CSS
 - JavaScript
 
-## Screenshot
+## My Demo
+[live demo](https://www.loom.com/share/ffe0947cbb05410eae7b9009dc99345b)
 
```

Comment:
As requested by the previous reviewer:
![image](https://user-images.githubusercontent.com/42799579/198194898-0cbcdaf7-1595-405f-b6d0-58b5fadc6ecb.png)

Your current project link is broken:
![image](https://user-images.githubusercontent.com/42799579/198195045-9328f11c-93db-4a29-903b-a4d6bf283b95.png)

Please make sure to add the correct one to your readme file.

Also, kindly update your 'Acknowledgments' section and give the design credit to [Cindy Shin](https://www.behance.net/adagio07) under [Creative Commons License](https://creativecommons.org/licenses/).